src/pages/transparency.tsx

```diff
@@ -47,140 +53,173 @@ export default function WrappingPage() {
   const l = useFormatMessage()
   const [data] = useAsyncMemo(async () => DclData.get().getData())
   const balances = useMemo(() => data && aggregateBalances(data.balances) || [], [data])
+  const burgerShow = useContext(BurgerMenuShowContext)
+  const burgerMenu = useContext(BurgerMenuStatusContext)
+  const responsive = useResponsive()
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth })
+  const translate = '100px'
+
+  useEffect(() => {
```

Comment:
This useEffect is being reused, maybe it can has its own file.

Reply:
+1